.github/workflows/release.yml

```
@@ -48,3 +48,8 @@ jobs:
           from_branch: main
           target_branch: dev
           github_token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+      - name: "Publish to NPM"
+        uses: JS-DevTools/npm-publish@v1
+        with:
+          token: ${{ secrets.NPM_TOKEN }}
```

@sherakama Can you confirm this is the correct environment variable name? (I don't have permissions to view the environment variables in Github, but this was what was being used in Decanter, so seemed like a good guess).

---

```suggestion
          token: ${{ secrets.NPMJS_AUTH_TOKEN }}
```

Ah, sorry for the naming discrepancy.